dev/run-tests.py

```diff
@@ -636,6 +636,7 @@ def _test():
     if failure_count:
         sys.exit(-1)
 
+
```

Comment:
Let's remove this newline.

Reply:
Hi @HyukjinKwon 

I add the new line to solve **PEP8: expected 2 blank lines after class or function definition, found 1**, do you suggest we ignore the PEP8 warning here?

Reply:
If it's for PEP8, keep it.

Reply:
Yea that's fine. I wonder why it didn't fail by pep8 check before tho.